modules/data.remote/R/call_MODIS.R

```diff
@@ -111,11 +164,33 @@ call_MODIS <- function(outfolder = ".", start_date, end_date, lat, lon, size = 0
     output$lat <- round(output$lat, 4)
     output$lon <- round(output$lon, 4)
     
-    fname <- paste(product, "_", band, "output_", start_date, "_", end_date, "_", lat, "_", lon, ".csv", sep = "")
-    fname <- paste0(outfolder, "/", fname)
-    write.csv(output, fname)
-    return(output)}
-
+    if (QC_filter == T)
+    {
+      output$qc == as.character(output$qc)
+      for (i in 1:nrow(output))
```

Comment:
`seq_len` is safer for the edge case where `nrow(output) == 0`:
```suggestion
      for (i in seq_len(nrow(output)))
```

Reply:
please update @bailsofhay